keps/693-multikueue/README.md

```diff
@@ -0,0 +1,262 @@
+MultiKueuCluster status as well as AdmissionCheckStatus and Events. MultiKueue controller 
+will make sure that whenever the kubeconfig is refreshed, the appropriate 
+clients will also be recreated.
+
+Creation of kubeconfig files is outside of the MultiKueue scope, and is cloud
+provider/environment dependant.
+
+MultiKueue controller, when pushing workloads to the worker clusters, will use the same 
+namespace and local queue names as were used in the management cluster. It is user's 
+responsibility to set up the appropriate namespaces and local queues.
+Worker ClusterQueue definitions may be different than in the management cluster. For example,
+quota settings may be specific to the given location. And/or cluster queue may have different
+admission checks, use ProvisioningRequest, etc.
+
+When distributing the workloads across clusters MultiKueue controller will first create
+the Kueue's internal Workload object. Only after the workload is admitted and other clusters
+are cleaned-up the real job will be created, to match the Workload. That gives the guarantee
+that the workload will not start in more than one cluster. The workload will
+get the annotation stating where it is actually running.
+
+When the job is running MultiKueue controller will copy its status from worker cluster
+to the management cluster, to keep the impression that the job is running in the management 
+cluster. This is needed to allow pipelines and workflow engines to execute against 
+the management cluster with MultiKueue without any extra changes. 
+
+#### Follow ups ideas
+
+* Handle large number of clusters via selectors.
+* Provide plugin mechanism to controll how the workloads are distributed across worker clusers.
+
+### Test Plan
+[x] I/we understand the owners of the involved components may require updates to
+existing tests to make this code solid enough prior to committing the changes necessary
+to implement this enhancement.
+
+#### Unit Tests
+The code will adhere to regular best practices for unit tests and coverage. 
+
+#### Integration tests
+Integration tests will be executed against a mocked clients for the worker clusters 
+that will provide predefined responses and allow to test various error scenarios, 
+including situations like:
+
+* Job is created across multiple clusters and admitted in one.
+* Job is admitted at the same time by two clusters.
+* Job is rejected by a cluster.
+* Worker cluster doesn't have the corresponding namespace.
+* Worker cluster doesn't have the corresponding local/cluster queue.
+* Worker cluster is unresponsive.
+* Worker cluster deletes the job.
+* Job is correctly finished.
+* Job finishes with an error.
+* Job status changes frequently.
+
+#### E2E tests
+Should be created and cover similar use cases as integration tests. For start
+it should focus on JobSet.
+
+### Graduation Criteria
+The feature starts at the alpha level, with a feature gate.
```

Comment:
Can you add some bullet points for what will make it to the 0.6 release as alpha?

For example, that only JobSet might be supported.

Reply:
Added.